src/medCore/gui/commonWidgets/medSlider.cpp

```diff
@@ -15,25 +15,36 @@
 
 void medSlider::addTick(int position)
 {
-    if(!ticksList.contains(position))
+    if( (tickCount() == 0) || !ticksList.contains(position))
```

Comment:
aaaaaah ok. This is not a Qt recommendation problem: in `tickCount()` there is a `tickList.isEmpty()` test to prevent crash. 
But i'm ok with your suggestion. Less computations. It's better in these cases :)